python/dgl/data/lrgb.py

```diff
@@ -0,0 +1,232 @@
+import hashlib
+import os
+import pickle
+
+import pandas as pd
+from ogb.utils import smiles2graph
+from tqdm import tqdm
+
+from .. import backend as F
+
+from ..convert import graph as dgl_graph
+from .dgl_dataset import DGLDataset
+from .utils import download, load_graphs, save_graphs, Subset
+
+
+class PeptidesStructuralDataset(DGLDataset):
```

Comment:
for all python documentation:
Max: 80 chars including heading space, and also try to make full use of the 80 chars as much as possible.